frontend/tests/snapshots/search.test.tsx

```diff
@@ -7,8 +7,11 @@ jest.mock("../../compositions/visualizations/map")
 beforeAll(() => setAuthForTest())
 
 it("renders Dashboard correctly", async () => {
+  // Ensure consistent title id's for fontawesome
```

Comment:
The tests still seem to pass with these lines removed. Could you revert the changes in this file?

Reply:
resolved in db449aa